app/web-app/src/app/modules/home/pages/shopping-lists/shopping-lists.component.ts

```diff
@@ -57,45 +62,62 @@ export class ShoppingListsComponent implements OnInit {
   }
 
   getShoppingList(id) {
-    console.log('get shopping list');
     this.shoppingListStore.setSelectedListById(id);
+    this.currentList = this.shoppingListStore.getSelectedList();
```

Comment:
Although getSelectedList() exists and returns a shopping list, it is bad practice for it to be used this way.
Consider making the code more reactive, by piping the result of other observables in this component to reflect changes